src/main/java/io/appform/dropwizard/actors/base/UnmanagedConsumer.java

```diff
@@ -68,24 +66,27 @@ public UnmanagedConsumer(final String name,
 
     public void start() throws Exception {
         for (int i = 1; i <= config.getConcurrency(); i++) {
-            Channel consumeChannel = connection.newChannel();
-            final Handler<Message> handler =
-                    new Handler<>(consumeChannel, mapper, clazz, prefetchCount, errorCheckFunction, retryStrategy,
-                                  exceptionHandler, handlerFunction, expiredMessageHandlingFunction, observer, queueName);
             String queueNameForConsumption;
             if (config.isSharded()) {
                 queueNameForConsumption = NamingUtils.getShardedQueueName(queueName, i % config.getShardCount());
             } else {
                 queueNameForConsumption = queueName;
             }
-
-            final String tag = consumeChannel.basicConsume(queueNameForConsumption, false, getConsumerTag(i), handler);
-            handler.setTag(tag);
-            handlers.add(handler);
-            log.info("Started consumer {} of type {} with tag {}", i, name, tag);
+            createChannel(queueNameForConsumption);
         }
     }
 
+    private void createChannel(String queueName) throws Exception {
```

Comment:
move private functions below public and private static

Reply:
Done